src/components/Home/ClusterDashboardItem.js

```diff
@@ -251,6 +254,13 @@ function ClusterDashboardItem({
               </span>
             </RefreshableLabel>
             <Dot style={{ paddingLeft: 0 }} />
+            <RefreshableLabel value={release.kubernetesVersion}>
+              <span>
+                <i className='fa fa-kubernetes' title='Kubernetes version' />{' '}
+                {release.kubernetesVersion}
```

Comment:
However, inside the `span` element, you might want to add a placeholder

```suggestion
                <i className='fa fa-kubernetes' title='Kubernetes version' />{' '}
                {release?.kubernetesVersion ?? 'N/A'}
```

Reply:
Done